tests/integration/component-behavior-test.js

```diff
@@ -26,7 +26,7 @@ module('Integration | Component Behavior', function(hooks) {
 
```

Comment:
We should probably add some tests that cover inheritance behavior, to make sure that it works correctly.

Reply:
The tests in `tests/unit/argument-test.js` should cover the inheritance -- there are a number of examples there around extending classes with `@argument` defined in a base class and overwritten (or not) in a parent class.